pkg/api/service_cache.go

```diff
@@ -92,7 +96,7 @@ func (c *ServiceCache) Refresh(ctx context.Context) error {
 	begin := time.Now()
 
 	var (
-		uri     = "https://api.fastly.com/service?page=1&per_page=100"
+		uri     = fmt.Sprintf("https://api.fastly.com/service?page=1&per_page=%d", maxServicePageSize)
```

Comment:
Why not just hard-code it like before?

Reply:
Just personal style preference, I like the explicitness and visibility for other readers of a constant for values like this. Very happy to have inline if you wish?

Reply:
That's fine! Just wondering if there was another reason.